index.html

```diff
@@ -9,6 +9,19 @@
     <link rel="stylesheet" href="css/style.css" />
   </head>
   <body>
+
```

Comment:
You've got a lot of `id's in this document. I would advice to think if you need all of them. You can style all of these elements using only classes and tag selectors, and use `id`s only to reach certain elements with JS.

It's more a case of good practise to style elements by using different selectors, as `id`s have a very strong rule (high [specificity](http://www.standardista.com/css3/css-specificity/) value, read on it, when you have a time) and using them when you don't really need to can make styles hard to maintain.